debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -234,6 +248,13 @@ private static int validateMaxQueueSize(Configuration config, Field field, Field
         return count;
     }
 
+    private static boolean isUsingAvroConverter(Configuration config) {
```

Comment:
Will this work if `AvroConverter` will be configured at worker level? Even if it does do we really want to apply the code automatically?

Reply:
@jpechane, nope the auto-enable would only work if enabled at the connector level.  For the worker style of configuration, manual enabling would be required.